api/server/graph.cc

```diff
@@ -639,9 +649,19 @@ void Graph::fw_update(const app::Nic &nic) {
             it++;
             continue;
         }
-        in_rules += "( " + fw_build_sg(sit->second) + " )";
+        auto sg_rules = fw_build_sg(sit->second);
```

Comment:
same here, please don't use auto, when the type is not explicit, I mean in a for with an iterator, the type is pretty explicit, when follow by a new, the type is pretty explicit too, but here, I need to fw_build_sg if I want to understand what sg_rules is, and that make the code harder to read.